vic/drivers/shared_image/src/state_metadata.c

```diff
@@ -0,0 +1,676 @@
+           "STATE_LAKE_ICE_AREA");
+    strcpy(state_metadata[STATE_LAKE_ICE_AREA_FRAC].long_name, "lake_ice_area");
+    strcpy(state_metadata[STATE_LAKE_ICE_AREA_FRAC].standard_name,
+           "lake_ice_coverage");
+    strcpy(state_metadata[STATE_LAKE_ICE_AREA_FRAC].units, "m2");
+    strcpy(state_metadata[STATE_LAKE_ICE_AREA_FRAC].description,
+           "area of ice coverage on lake at beginning of time step");
+
+    // STATE_LAKE_ICE_AREA_FRAC_NEW
+    strcpy(state_metadata[STATE_LAKE_ICE_AREA_FRAC_NEW].varname,
+           "STATE_LAKE_ICE_AREA_NEW");
+    strcpy(state_metadata[STATE_LAKE_ICE_AREA_FRAC_NEW].long_name,
+           "lake_ice_area_new");
+    strcpy(state_metadata[STATE_LAKE_ICE_AREA_FRAC_NEW].standard_name,
+           "lake_ice_area_new");
+    strcpy(state_metadata[STATE_LAKE_ICE_AREA_FRAC_NEW].units, "m2");
+    strcpy(state_metadata[STATE_LAKE_ICE_AREA_FRAC_NEW].description,
+           "area of ice coverage on lake at end of time step");
+
+    // STATE_LAKE_ICE_WATER_EQUIVALENT
+    strcpy(state_metadata[STATE_LAKE_ICE_WATER_EQUIVALENT].varname,
+           "STATE_LAKE_ICE_WATER_EQUIVALENT");
+    strcpy(state_metadata[STATE_LAKE_ICE_WATER_EQUIVALENT].long_name,
+           "lake_ice_water_equivalent");
+    strcpy(state_metadata[STATE_LAKE_ICE_WATER_EQUIVALENT].standard_name,
+           "lake_ice_water_equivalent");
+    strcpy(state_metadata[STATE_LAKE_ICE_WATER_EQUIVALENT].units, "m3");
+    strcpy(state_metadata[STATE_LAKE_ICE_WATER_EQUIVALENT].description,
+           "liquid water equivalent volume of lake ice");
+
+    // STATE_LAKE_ICE_HEIGHT
+    strcpy(state_metadata[STATE_LAKE_ICE_HEIGHT].varname,
+           "STATE_LAKE_ICE_HEIGHT");
+    strcpy(state_metadata[STATE_LAKE_ICE_HEIGHT].long_name, "lake_ice_height");
+    strcpy(state_metadata[STATE_LAKE_ICE_HEIGHT].standard_name,
+           "lake_ice_height_thickest");
+    strcpy(state_metadata[STATE_LAKE_ICE_HEIGHT].units, "m");
+    strcpy(state_metadata[STATE_LAKE_ICE_HEIGHT].description,
+           "lake ice height at ghickest point");
+
+    // STATE_LAKE_ICE_TEMP
+    strcpy(state_metadata[STATE_LAKE_ICE_TEMP].varname, "STATE_LAKE_ICE_TEMP");
+    strcpy(state_metadata[STATE_LAKE_ICE_TEMP].long_name, "lake_ice_temp");
+    strcpy(state_metadata[STATE_LAKE_ICE_TEMP].standard_name,
+           "lake_ice_temperature");
+    strcpy(state_metadata[STATE_LAKE_ICE_TEMP].units, "C");
+    strcpy(state_metadata[STATE_LAKE_ICE_TEMP].description,
+           "lake ice temperature");
+
+    // STATE_LAKE_ICE_SWE
+    strcpy(state_metadata[STATE_LAKE_ICE_SWE].varname, "STATE_LAKE_ICE_SWE");
+    strcpy(state_metadata[STATE_LAKE_ICE_SWE].long_name, "lake_ice_swe");
+    strcpy(state_metadata[STATE_LAKE_ICE_SWE].standard_name,
+           "lake_snow_water_equivalent");
+    strcpy(state_metadata[STATE_LAKE_ICE_SWE].units, "m");
+    strcpy(state_metadata[STATE_LAKE_ICE_SWE].description,
+           "liquid water equivalent depth of lake snow");
+
```

Comment:
I'm not sure if all these lake variables are necessary to be saved in the state file (e.g., not sure how this variable `STATE_LAKE_ICE_SWE` is different from `STATE_LAKE_SNOW_WATER_EQUIVALENT` above). It might be good if @tbohn could take a look at the lake state section (or comment somewhere and leave careful investigation of the lake states to the future)?